lib/ardb/runner/create_command.rb

```diff
@@ -3,19 +3,21 @@
 
 class Ardb::Runner::CreateCommand
 
-  def initialize
+  def initialize(out_io = nil, err_io = nil)
+    @out_io = out_io || $stdout
+    @err_io = err_io || $stderr
     @adapter = Ardb::Adapter.send(Ardb.config.db.adapter)
   end
 
   def run
     begin
       @adapter.create_db
-      $stdout.puts "created #{Ardb.config.db.adapter} db `#{Ardb.config.db.database}`"
+      @out_io.puts "created #{Ardb.config.db.adapter} db `#{Ardb.config.db.database}`"
     rescue Ardb::Runner::CmdError => e
       raise e
     rescue Exception => e
```

Comment:
@Criptak - while you are here can you switch these (and all the other runners) to rescue `StandardError` instead?